website/redirects.js

```diff
@@ -50,8 +50,8 @@ module.exports = [
     permanent: true,
   },
   {
-    source: '/docs/auth/jwt_oidc_providers',
-    destination: '/docs/auth/jwt/oidc_providers',
+    source: '/docs/auth/jwt/oidc_providers',
```

Comment:
I tried to fix all of our own links. I'm wondering if I should add additional redirects from the old anchors to the new pages.

Reply:
Hmm, I think adding redirects for each section anchor could be overkill. We should be able to add them as needed if we get a request to do so.

I tested the redirect here using the deploy preview. It works as expected.